packages/ra-core/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "ra-core",
+    "version": "2.0.0-beta2",
+    "description": "Core components of react-admin, a frontend Framework for building admin applications on top of REST services, using ES6, React",
+    "files": [
+        "*.md",
+        "docs/*.md",
+        "lib",
+        "src"
+    ],
+    "main": "lib/index",
+    "authors": [
+        "François Zaninotto",
+        "Gildas Garcia"
+    ],
+    "repository": "marmelab/react-admin",
+    "homepage": "https://github.com/marmelab/react-admin#readme",
+    "bugs": "https://github.com/marmelab/react-admin/issues",
+    "license": "MIT",
+    "devDependencies": {
+        "enzyme": "~3.1.0",
+        "enzyme-adapter-react-16": "~1.0.1",
+        "file-api": "~0.10.4",
+        "full-icu": "~1.1.3",
+        "ignore-styles": "~5.0.1",
+        "react": "~16.0.0",
+        "react-dom": "~16.0.0",
+        "react-test-renderer": "~16.0.0"
+    },
+    "peerDependencies": {
+        "react": "^15.4.0 || ~16.0.0",
+        "react-dom": "^15.4.0 || ~16.0.0"
+    },
+    "dependencies": {
+        "babel-runtime": "~6.26.0",
+        "classnames": "~2.2.5",
+        "inflection": "~1.12.0",
+        "lodash.debounce": "~4.0.8",
+        "lodash.defaultsdeep": "~4.6.0",
+        "lodash.get": "~4.4.2",
+        "lodash.pickby": "~4.6.0",
+        "lodash.set": "~4.3.2",
+        "lodash.uniq": "~4.5.0",
+        "node-polyglot": "2.2.2",
+        "prop-types": "~15.5.7",
+        "query-string": "~5.0.0",
+        "ra-language-english": "^2.0.0-beta2",
```

Comment:
should not be used here, only in `react-admin`

Reply:
It's the default used by the `TranslationProvider`, which is provided by `ra-core`

Reply:
:+1: